src/test/java/com/google/api/codegen/testdata/discogapic/java/java_fakebilling.v1.json.baseline

```diff
@@ -337,19 +337,18 @@ public final class BillingAccount implements Serializable {
 package com.google.cloud.fakebilling.v1;
 
 import com.google.api.core.BetaApi;
-import com.google.common.collect.ImmutableListMultimap;
 import java.util.LinkedList;
 import java.util.List;
+import java.util.Map;
+import java.util.Set;
 import javax.annotation.Generated;
 
 @Generated("by GAPIC")
 @BetaApi
-public final class BillingAccountsGetHttpRequest {
+public final class BillingAccountsGetHttpRequest implements Message {
```

Comment:
I think we should use a more unique class name for the message interface - I suggest `ApiMessage`